jupyter-config.yml

```diff
@@ -27,14 +27,13 @@ pangeo:
               )
               return pod
           c.KubeSpawner.modify_pod_hook = modify_pod_hook
-
-  # The templates section is borrowed from the pangeo approach, i.e.
-  # https://github.com/pangeo-data/pangeo-cloud-federation/blob/staging/deployments/ocean/config/common.yaml
-  # TODO: Maybe just merge this template into the notebook image repo and do the
-  # configuration there rather than maintaining a separate repo and mounting it
-  # in this deploy config?
           c.JupyterHub.logo_file = '/usr/local/share/jupyter/hub/static/custom/images/logo.png'
           c.JupyterHub.template_paths = ['/usr/local/share/jupyter/hub/custom_templates/']
+# The templates section is borrowed from the pangeo approach, i.e.
+# https://github.com/pangeo-data/pangeo-cloud-federation/blob/staging/deployments/ocean/config/common.yaml
+# TODO: Maybe just merge this template into the notebook image repo and do the
+# configuration there rather than maintaining a separate repo and mounting it
+# in this deploy config?
```

Comment:
This wouldn't work. these templates affect the hub, not the notebook.
the entire templates repo is mounted on the hub pod and is used to override
the jupyterhub _hub_ web jinja templates. this doesn't have anything to do
with the notebook/worker images, except in that bad jinja can result in
interface failures

Reply:
Ah yes Good point